cpp/modmesh/view/RAction.hpp

```diff
@@ -47,43 +47,6 @@ class RAction
         QObject * parent = nullptr);
 }; /* end class RAction */
 
-class RPythonAction
```

Comment:
With action setup code ported to Python, this helper class `RPythonAction` is not longer needed.